fbfmaproom/fbflayout.py

```diff
@@ -403,87 +403,28 @@ def table_layout():
             dcc.Loading(
                 [
                     table.DataTable(
-                        id="summary",
+                        id="table",
                         page_action="none",
                         style_table={
-                            "height": "auto",
-                            "overflowY": "scroll",
+                            "height": "900px",
```

Comment:
The height of the table needs to be increased since it encompasses both the header and the table proper

Reply:
Yeah, I remember looking into that and not finding a solution. This is one of the things we planned to address in switching to a new layout.

Reply:
[Here's a discussion of this issue.](https://community.plotly.com/t/setting-datatable-max-height-when-using-fixed-headers/26417) It appears to be a known problem.

Reply:
I was able to increase the size by simply setting the height and max height to the same value.